tests/integration/new_relations/test_new_relations.py

```diff
@@ -187,7 +187,7 @@ async def test_two_applications_doesnt_share_the_same_relation_data(
     # Check that the user cannot access other databases.
     for application, other_application_database in [
         (APPLICATION_APP_NAME, "another_application_first_database"),
-        (another_application_app_name, "application_first_database"),
+        (another_application_app_name, f"{APPLICATION_APP_NAME.replace('-', '_')}_first_database"),
```

Comment:
Using the first interface for HA tests, Since continuous_writes is started on demand, it shouldn't affect relation testing.